go-controller/pkg/metrics/node.go

```diff
@@ -34,9 +34,20 @@ var metricOvnNodePortEnabled = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "Specifies if the node port is enabled on this node(1) or not(0).",
 })
 
+// metric to get the size of ovnkube.log file
+var metricOvnKubeNodeLogFileSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemNode,
+	Name:      "logfile_size",
+	Help:      "The size of ovnkube logfile on the node."},
+	[]string{
+		"logfile_name",
```

Comment:
`log_file_name`

Reply:
logfile is a common usage elsewhere in kubernetes code, so going to keep what I have.